doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -994,7 +994,7 @@
 
     std::string myStdString("foo");
     // std::string to iox::string
-    iox::string<3> myIoxString = iox::into<iox::string<3>>(myStdString);
+    iox::string<3> myIoxString = iox::into<iox::lossy<iox::string<3>>>(myStdString);
```

Comment:
Small nitpick: In the `iox::string` we talk about "truncation". For an new user it might not be clear that there is no difference between truncation and lossy. Maybe we should use the term truncation also here?

Reply:
@mossmaurice but then it would be weird to use it for other types beside the string which could also have a lossy conversion. I'm not fully convinced it is the better name.

Reply:
True, for other types 'truncation' does not make sense. Let's stick to lossy then.